src/core/chart-api/chart-extra-legend.tsx

```diff
@@ -82,8 +82,8 @@ export class ChartExtraLegend extends AsyncStore<ReactiveLegendState> {
 
   private initLegend = () => {
     const itemSpecs = getChartLegendItems(this.context.chart());
-    const legendItems = itemSpecs.map(({ id, name, color, markerType, visible }) => {
-      const marker = this.renderMarker(markerType, color, visible);
+    const legendItems = itemSpecs.map(({ id, name, color, markerType, visible, status }) => {
+      const marker = this.renderMarker(markerType, color, visible, status);
```

Comment:
We need to have unit tests for this.

Also, we might consider adding additional test utils method(s) to find legend items with warning - this should be helpful for unit tests on your end, so that you don't have to depend on the impl detail.